pennylane/devices/default_qubit.py

```diff
@@ -185,6 +186,25 @@ def __init__(
             "Toffoli": self._apply_toffoli,
         }
 
+    @property
+    def stopping_condition(self):
+        def accepts_obj(obj):
+            if getattr(obj, "has_matrix", False):
+                # pow operations dont work with backprop or adjoint without decomposition
+                # use class name string so we don't need to use isisntance check
+                if obj.__class__.__name__ == "Pow" and qml.operation.is_trainable(obj):
+                    return False
+                return True
```

Comment:
Fly-by comment: could we do this? (Or whatever the eventual boolean expression might be)
```suggestion
                    return obj.__class__.__name__ == "Pow" and qml.operation.is_trainable(obj)
```

Reply:
```suggestion
               return not (obj.__class__.__name__ == "Pow" and qml.operation.is_trainable(obj)):
```